### DISCUSSION TITLE
Does Select support search by keywords?

### DISCUSSION THREAD

**Discussion by Tim-Cao** [Textualize/textual#3590] [Q&A] [answered]
Image there are many options for a select field. Does the Select support search by keywords?
![image](https://github.com/Textualize/textual/assets/52661397/1e60127a-6ec0-4e9b-8291-d74e100ebfcb)

**Comment by davep** [comment, marked as answer]
No, Select doesn't have that ability. You can read what it can do in the docs: https://textual.textualize.io/widgets/select/

**Reply by Tim-Cao** [reply]
Is it on roadmap?

**Reply by davep** [reply]
I don't know of any plans internally to add that sort of support to `Select` any time soon.

**Reply by Tim-Cao** [reply]
Thanks
